### ISSUE TITLE
SyntaxError on attempt to run

### ISSUE THREAD

**Issue by cboden** [adonisjs/require-ts#1]
## Package version
1.0.0

## Node.js and npm version
10.17.0 and 6.12.1 (yarn version 1.19.1)

## BONUS (a sample repo to reproduce the issue)
https://github.com/cboden/require-ts-test

I get an error when attempting to try this on an existing project. I broke out the fundamental issue for replication purposes in a new repository. I receive the following error:

```
/Users/cboden/Repositories/request-ts-test/src/node_modules/MyPackage/index.ts:1
import {format} from 'util';
       ^

SyntaxError: Unexpected token {
    at Module._compile (internal/modules/cjs/loader.js:723:23)
    at Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Object.newLoader [as .ts] (/Users/cboden/Repositories/request-ts-test/node_modules/pirates/lib/index.js:104:7)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/Users/cboden/Repositories/request-ts-test/src/index.ts:1:1)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
```

**Comment by thetutlage** [comment]
Ahh got it. It is because you are trying to cosume a node_module written in Typescript. The package is meant to parse the source files within the project and expects the node_modules to be always published in Javascript.

**Comment by cboden** [comment]
Is this feature patchable in `require-ts`? The module is within my own src project, not a published package. It would make sense to me that it would be traversed by the parser.

**Comment by thetutlage** [comment]
> Is this feature patchable in require-ts

Nope, coz even Typescript itself recommends compiling Typescript to Javascript when publishing modules.

I understand it is not published, but the import resolution is not of a local file, but of a module.

- `import something from './something'` is a local import
- `import something from 'something'` is a module import from the node_modules and it doesn't matter if the module was published on npm or not.

In other words. The package is not meant to hook into the imports resolved as node_modules depndencies. Since it will make everything super slow, if we begin compiling typescript source code within the node_modules

**Comment by cboden** [comment]
> the import resolution is not of a local file, but of a module

Ah damn. That makes sense to me now.

**Comment by thetutlage** [comment]
Nice. I will close the issue then 😄
